packages/core/src/app/payment/paymentMethod/CreditCardPaymentMethod.tsx

```diff
@@ -89,6 +90,7 @@ class CreditCardPaymentMethod extends Component<
 > {
     state: CreditCardPaymentMethodState = {
         isAddingNewCard: false,
+        isPreloaderOn: true,
```

Comment:
This component is used by a quite a few providers, have you tested if the other providers work fine?

Reply:
This state sets to false whenever any of providers finish initialization, so should work fine for any provider that uses this component. I've checked PaypalCommerce and Braintree

Reply:
Can you please test QP access and Hosted credit card as well. Thanks.

Reply:
What do you mean by WP access? Could you explain please

Reply:
@animesh1987 I've tested PaypalCommerce, Braintree and regular Credit Card with no provider onboarded